frontend/src/shared/helpers.tsx

```diff
@@ -475,13 +476,15 @@ function setupEditableSampleFields(samplesColDefs: ColDef[]) {
         });
         return changedValue !== undefined;
       },
-      "costCenter-validation-error": (params: CellClassParams) => {
-        if (params.colDef.field === "costCenter") {
-          return !isValidCostCenter(params.value);
-        }
-        return false;
-      },
```

Comment:
Moved this CSS class setup from inside this function to `CohortSampleDetailsColumns`. The reason is to keep this function strictly for setting up editable fields (per its name `setupEditableSampleFields`).